components/x11/xorg-server/files/x11-init.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+ CDDL HEADER START
+
+ The contents of this file are subject to the terms of the
+ Common Development and Distribution License (the "License").
+ You may not use this file except in compliance with the License.
+
+ You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+ or http://www.opensolaris.org/os/licensing.
+ See the License for the specific language governing permissions
+ and limitations under the License.
+
+ When distributing Covered Code, include this CDDL HEADER in each
+ file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+ If applicable, add the following below this CDDL HEADER, with the
+ fields enclosed by brackets "[]" replaced with your own identifying
+ information: Portions Copyright [yyyy] [name of copyright owner]
+
+ CDDL HEADER END
+
+	NOTE:  This service manifest is not editable; its contents will
+	be overwritten by package or patch operations, including
+	operating system upgrade.  Make customizations in a different
+	file.
+
+	Service manifest for the DNSCrypt proxy service.
+-->
+<!--
+    Copyright (c) 2011, Oracle and/or its affiliates. All rights reserved.
+    Copyright 2019 Alexander Pyhalov
+-->
+<service_bundle type='manifest' name='x11-init'>
+
+<service name='application/x11/x11-init' type='service' version='1'>
+	
+	<create_default_instance enabled='true' />
+
+	<single_instance/>
+
+	<dependency
+            name='filesystem'
+            grouping='require_all'
+            restart_on='none'
+            type='service'>
+                <service_fmri value='svc:/system/filesystem/local'/>
+        </dependency>
+
+        <dependency name='single-user'
+            grouping='require_all'
+            restart_on='error'
+            type='service'>
+                <service_fmri value='svc:/milestone/single-user:default'/>
+        </dependency>
```

Comment:
Indentation is jumpy, and should this be required by an X11 service, e.g. lightdm, to surely start before?

Reply:
Ok, I see the dependency is covered from the other side in #5328